winui/Avatar-view/visual-styles.md

```diff
@@ -1,10 +1,364 @@
 ---
 layout: post
 title: Visual Styles in WinUI AvatarView control | Syncfusion
-description: This section will explain about the different types of visual styles that exists in WinUI AvatarView.
+description: Learn here all about the different visual styles that are available in the WinUI AvatarView.
 platform: WinUI
 control: AvatarView
 documentation: ug
 ---
 
-# Visual Styles in WinUI AvatarView
\ No newline at end of file
+# Visual Styles in WinUI AvatarView
+
+The `AvatarView` control provides support for various built-in visual styles. You can configure the visual style of your AvatarView by using the `AvatarShape` property and select from the available options, such as:
+
+* Custom
+* Circle
+* Square
+
+## Custom
+
+Custom type allows you to customize the AvatarView control with height, width, font and corner radius.
```

Comment:
The custom type allows you to customize the AvatarView control with height, width, font, and corner radius.